io/disk/dbench.py.data/dbench.yaml

```diff
@@ -11,5 +12,22 @@ setup:
             nprocs: 1
     location:
         default:
```

Comment:
@smruti77 you need to remove these lines as well.
And make sure Readme is also updated, if needed, in all files.